spec/folio_client/authenticator_spec.rb

```diff
@@ -73,6 +73,15 @@
       end
     end
 
+    context "when the service returns a 409 conflict error" do
+      let(:http_status) { 409 }
+      let(:http_body) { "{\"error\" : \"get bent\"}" }
+
+      it "raises a service unavailable exception" do
```

Comment:
copypasta?

Reply:
🤦‍♀️ yes, fixed!

Reply:
something something stones and glass houses